oauth2client/crypt.py

```diff
@@ -37,7 +39,10 @@ class AppIdentityError(Exception):
 
 
 try:
-  from OpenSSL import crypto
+  _, package_dir, _ = imp.find_module('OpenSSL')
```

Comment:
maybe just call it `_package_dir`, and not worry about deleting it?

(in particular, i think the variable hangs around in the event of the exception two lines down -- so if you want to leave it public, maybe `del` there, too?)